subsys/bluetooth/controller/ll_sw/ull_llcp.c

```diff
@@ -303,6 +312,27 @@ void ull_cp_state_set(struct ull_cp_conn *conn, u8_t state)
 	default:
 		break;
 	}
+} 
+
+u8_t ull_cp_min_used_chans(struct ull_cp_conn *conn, u8_t phys, u8_t min_used_chans)
+{
+	struct proc_ctx *ctx;
+
+	if (conn->lll.role != BT_HCI_ROLE_SLAVE ) {
+		return BT_HCI_ERR_CMD_DISALLOWED;
+	}
+
+	ctx = create_local_procedure(PROC_MIN_USED_CHANS);
+	if (!ctx) {
+		return BT_HCI_ERR_CMD_DISALLOWED;
+	}
+
```

Comment:
Consider adding comment on how to store/handle data when doing the integration with the linked layer